kiro-cli/kiro-cli-config/prompts/start.md

```diff
@@ -0,0 +1,18 @@
+You are an expert in OpenSearch Migration Assistant. Help me migrate data from Elasticsearch/OpenSearch to OpenSearch.
+
+I need to migrate:
+- Source: [ENDPOINT] ([VERSION], [AUTH_TYPE: SigV4|basic], [REGION])
+- Target: [ENDPOINT] ([VERSION], [AUTH_TYPE: SigV4|basic], [REGION])
+
+Options:
+- Indices: [all | specific patterns like logs-*, metrics-*]
+- Skip approvals: [yes | no]
+
+Please:
+1. Verify connectivity to both clusters
+2. Get S3/snapshot config from EKS configmap
+3. Check target state and show existing indices before proceeding
+4. Calculate podReplicas sizing and estimate migration time
+5. Generate the workflow configuration
+6. Submit and monitor the migration
+7. Verify doc counts match after completion
```

Comment:
for future thought: two comments - this is somewhat redundant from the json above...

I'm a bit worried about the maintainability of this.  This is great for a first-cut/POC, but I'd love to see this grounded in the source code a bit more.  We've talked about adding more metadata/decorations to the workflow models, but this could be another good reason to do that.

In general, can we think about this task depending upon other projects and pulling the source or artifacts in to build derivative kiro/agent artifacts?